### DISCUSSION TITLE
MAUI iOS getting C# Stacktrace on crash

### DISCUSSION THREAD

**Discussion by mkshiblu** [dotnet/maui#26312] [Q&A]
Hi, I was wondering if it is possible to get the C# stacktrace when the release build of the iOS app crash without using crashanalytics platform such as AppCenter or Sentry?

I tried to symbolicate using the .DSYM file but it is showing only the swift strack trace not the C# one. Not sure how to link the PDB file though.

Wondering how AppCenter or other crashanalytics tool does produce this information, like do they do any kind of static analysis or just use some .NET tool to generate the report?

**Comment by stephenquan** [comment]
This discussion thread may help you https://github.com/dotnet/maui/discussions/653

**Reply by mkshiblu** [reply]
Hu @stephenquan, thanks for your reply. I did went through the discussion and could not find anything promising there. Any other tips?
